src/utils.ts

```diff
@@ -24,6 +26,23 @@ export function getCheckRunPayloadWithScanResult(trivyStatus: number, dockleStat
     return checkRunPayload;
 }
 
+export function getScanReportPath(): string {
+  const scanReportPath = `${fileHelper.getContainerScanDirectory()}/scanReport.json`;
+  return scanReportPath;
+}
+
+export function createScanReport() {
+  const scanReportPath = getScanReportPath();
+  const trivyOutput = trivyHelper.getTrivyFilteredOutput();
+  const dockleOutput = dockleHelper.getDockleFilteredOutput();
+  const scanReportObject = {
+    "commonVulnerabilities": trivyOutput,
+    "bestPracticeVulnerabilities": dockleOutput
+  };
+  const output = `{\n\t"commonVulnerabilities": ${trivyOutput},\n\t"bestPracticeVulnerabilities": ${dockleOutput}\n}`;
```

Comment:
Create a JSON and use `JSON.stringify`

Reply:
I'm already using JSON.stringify. This line was supposed to be deleted. I'll remove it.